key.ui/src/main/java/de/uka/ilkd/key/gui/extension/impl/KeYGuiExtensionFacade.java

```diff
@@ -82,6 +81,14 @@ public static void addExtensionsToMainMenu(MainWindow mainWindow, JMenuBar menuB
 
     }
 
+    /*
+     * public static Optional<Action> getMainMenuExtensions(String name) {
+     * Spliterator<KeYGuiExtension.MainMenu> iter =
+     * ServiceLoader.load(KeYGuiExtension.MainMenu.class).spliterator(); return
+     * StreamSupport.stream(iter, false) .flatMap(it -> it.getMainMenuActions(mainWindow).stream())
+     * .filter(Objects::nonNull) .filter(it -> it.getValue(Action.NAME).equals(name)) .findAny(); }
+     */
+
```

Comment:
```suggestion
```
Let's remove this code instead of having it commented out (this doesn't really help anybody).